README.md

```diff
@@ -1,49 +1,75 @@
 # GovWifi Authentication API
 
-Authentication API for GovWifi
-
-## Overview
+This API is used by the frontend when a connection request is made to GovWifi.
+Its function is to check these requests against user details and the result determines how FreeRadius reacts to it e.g. allow / dissallow the connection.
+
+The private GovWifi [build repository][build-repo] contains instructions on how to build GovWifi end-to-end - the sites, services and infrastructure.
+
+<!-- vscode-markdown-toc -->
+* 1. [Overview](#Overview)
+	* 1.1. [Sinatra routes](#Sinatraroutes)
+	* 1.2. [Dependencies](#Dependencies)
+* 2. [Developing](#Developing)
+	* 2.1. [Running the tests](#Runningthetests)
+	* 2.2. [Using the linter](#Usingthelinter)
+	* 2.3. [Serving the app locally](#Servingtheapplocally)
+	* 2.4. [Deploying changes](#Deployingchanges)
+* 3. [Gotchas](#Gotchas)
+	* 3.1. [Extra API parameters](#ExtraAPIparameters)
+* 4. [Licence](#Licence)
+
+<!-- vscode-markdown-toc-config
+	numbering=true
+	autoSave=true
+	/vscode-markdown-toc-config -->
+<!-- /vscode-markdown-toc -->
+
+##  1. <a name='Overview'></a>Overview
 
 GovWifi exposes FreeRADIUS servers on public IP addresses, which are configured
 to communicate with this private API via its REST plugin.
 
-### Sinatra routes
+###  1.1. <a name='Sinatraroutes'></a>Sinatra routes
 
 * `GET /authorize/user/:user_name`  - FreeRADIUS authorisation route
 
-### Dependencies
+###  1.2. <a name='Dependencies'></a>Dependencies
 
 * MySQL database - user details generated by the [user signup API][user-signup-api]
-  are fetched by this API 
+  are fetched by this API.
 
-[user-signup-api]: https://github.com/alphagov/govwifi-user-signup-api/pull/33
+##  2. <a name='Developing'></a>Developing
 
-## Developing
+###  2.1. <a name='Runningthetests'></a>Running the tests
 
-### Running the tests
+```shell
+make test
+```
 
-You can run the tests and linter with the following commands:
+###  2.2. <a name='Usingthelinter'></a>Using the linter
 
 ```shell
-make test
 make lint
 ```
 
-### Serving the app locally
+###  2.3. <a name='Servingtheapplocally'></a>Serving the app locally
 
 ```shell
 make serve
 ```
 
-### Deploying changes
+###  2.4. <a name='Deployingchanges'></a>Deploying changes
 
-Once you have merged your changes into master branch.  Deploying is made up of
-two steps.  Pushing a built image to the docker registry from Jenkins, and
-restarting the running tasks so it picks up the latest image.
+Once you have merged your changes into the master branch, deploying them is made up of
+two steps:
 
-## Gotchas
+* Pushing a built image to the docker registry from Jenkins.
 
-### Extra API parameters
+* Restarting the running tasks so it picks up the latest image.
+
+##  3. <a name='Gotchas'></a>Gotchas
```

Comment:
I'd remove this section entirely. Feels like a TODO note somewhere rather than something every reader of the README needs to know.

Suspect I'm not in the majority there though.